docs/user/openstack/README.md

```diff
@@ -105,63 +102,106 @@ openstack network list --long -c ID -c Name -c "Router Type"
+6. A name that identifies your cluster. Your api will be hosted at `<cluster name>.<base domain>`, and its resources will be prepended by `<cluster name>-<clusterID>-`
+7. The CIDR of the OpenStack subnet that your master and worker nodes will be hosted on
+8. OpenStack Region to install the OCP in
+9. OpenStack Cloud in clouds.yaml to use
+10. External network in OpenStack cluster to attach router to
+11. Flavor to use for Master Nodes. See the [OpenStack Requirements](#openstack-requirements) for details on requirements
+12. Floating IP assigned to the API load balancer
+13. The Pull secrete associated with your RHCOS image
+14. A public Key that will be injected onto all nodes in the cluster
+
+Once you create and customize your `install-config.yaml`, we recommend you create a directory for your cluster, and copy the config into it. See the documents on the [recommended workflow](../overview.md#multiple-invocations) for more information about why you should do it this way.
 
-Next, add the `api.<cluster name>.<cluster domain>` and `*.apps.<cluster
-name>.<cluster domain>` name records pointing to that floating IP to your DNS:
+```sh
+mkdir ostest
+cp install-config.yaml ostest/install-config.yaml
+```
 
-    api.example.shiftstack.com IN A <API FIP>
+## API Access
 
-If you don't have a DNS server under your control, you finish the installation
-by adding the following to your `/etc/hosts`:
+All the OpenShift nodes get created in an OpenStack tenant network and as such, can't be accessed directly in most OpenStack deployments. We will briefly explain how to set up access to the OpenShift API with and without floating IP addresses.
 
-    <API FIP> api.example.shiftstack.com
+### Using Floating IPs
 
-**NOTE:** *this will make the API accessible only to you. This is fine for your
-own testing (and it is enough for the installation to succeed), but it is not
-enough for a production deployment.*
+This method allows you to attach two floating IP (FIP) addresses to endpoints in OpenShift.
 
-Finally, add the floating IP address to `install-config.yaml`.
+You will need to create two floating IP addresses, one to attach to the API load balancer (lb FIP), and one for openshift apps (apps FIP). Note that the LB FIP is the same floating IP as the one you added to your `install-config.yaml`. The following command is an example of how to create floating IPs:
 
-It should be under `platform.openstack.lbFloatingIP`. For example:
+```sh
+openstack floating ip create <external network>
+```
 
-```yaml
-apiVersion: v1beta2
-baseDomain: shiftstack.com
-clusterID:  3f47f546-c010-4c46-895c-c8fce6cf0451
-# ...
-platform:
-  openstack:
-    cloud:            standalone
-    externalNetwork:  public
-    region:           regionOne
-    computeFlavor:    m1.large
-    lbFloatingIP:     "<API FIP>"
+You will also need to add the following records to your DNS:
+
+```dns
```

Comment:
This is `/etc/hosts` format, not DNS.
Also I think we should leave out the hosts resolving to <apps FIP> for now and document it below in the forgotten step 3.